### PR TITLE
Release/v1.0.15

### PR DESCRIPTION
**Pull request by platfowner**
New updates:
- Reduce the number of blockchain nodes to 5 from 10
- Disable peer whitelisting
- Use a default value of TRACKER_UPDATE_JSON_RPC_URL for testnet-prod 
- Add <Parent Node Index Begin> and <Parent Node Index End> options to deploy_blockchain_genesis_gcp.sh

### CODE REVIEW

File: blockchain-configs/testnet-prod/node_params.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
   "SNAPSHOTS_INTERVAL_BLOCK_NUMBER": 1000,
   "SYNC_MODE": "fast",
   "TARGET_NUM_OUTBOUND_CONNECTION": 3,
-  "TRACKER_UPDATE_JSON_RPC_URL": "NEED TO BE SET DIFFERENTLY FOR SPRING & SUMMER",
+  "TRACKER_UPDATE_JSON_RPC_URL": "http://35.221.137.80:8080/json-rpc",
```
Comment (ehgmsdk20) [inline blockchain-configs/testnet-prod/node_params.json:71]: question) is it okay to write the ip directly?

File: start_node_docker.sh [hunk 2]
```diff
@@ -3,9 +3,9 @@ if [[ $SEASON = 'mainnet' ]]; then
 elif [[ $SEASON = 'summer' ]]; then
     export BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/testnet-prod
     export TRACKER_UPDATE_JSON_RPC_URL="http://35.194.172.106:8080/json-rpc"
+    export PEER_CANDIDATE_JSON_RPC_URL="http://35.194.169.78:8080/json-rpc"
 elif [[ $SEASON = 'spring' ]]; then
     export BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/testnet-prod
-    export TRACKER_UPDATE_JSON_RPC_URL="http://35.221.137.80:8080/json-rpc"
```
Comment (ehgmsdk20) [inline start_node_docker.sh:8]: Why the TRACKER_UPDATE_JSON_RPC_URL is deleted from spring?

File: start_node_incremental_gcp.sh [hunk 3]
```diff
@@ -109,108 +120,90 @@ printf "\n#### [Step 1] Configure env vars ####\n\n"
...
-    else
-        export PEER_CANDIDATE_JSON_RPC_URL="http://104.199.237.250:8080/json-rpc"
     fi
 elif [[ $SEASON = 'summer' ]]; then
     export BLOCKCHAIN_CONFIGS_DIR=blockchain-configs/testnet-prod
-    export TRACKER_UPDATE_JSON_RPC_URL=http://35.194.172.106:8080/json-rpc
-    if [[ $NODE_INDEX -lt 5 ]]; then
-        export PEER_CANDIDATE_JSON_RPC_URL="https://summer-api.ainetwork.ai/json-rpc"
+    export TRACKER_UPDATE_JSON_RPC_URL="http://35.194.172.106:8080/json-rpc"
+    export PEER_CANDIDATE_JSON_RPC_URL="http://35.194.169.78:8080/json-rpc"
```
Comment (ehgmsdk20) [inline start_node_incremental_gcp.sh:129]: In mainnet, it seems that PEER_CANDIDATE_JSON_RPC_URL is not used anymore.
Is PEER_CANDIDATE_JSON_RPC_URL necessary for summer?

**Review by ehgmsdk20 — APPROVED** [review]
LGTM with some questions

File: blockchain-configs/testnet-prod/node_params.json (hunk 1 above)
Comment (platfowner) [reply blockchain-configs/testnet-prod/node_params.json:71]: We have two clusters (spring and summer) for the testnet-prod config, while we have only one cluster for other configs (e.g. mainnet, testnet-staging, etc). So we were not setting an explicit value for it. But for now we use only spring deprecating summer, so we decided to explicitly set the spring tracker's endpoint here.

File: start_node_docker.sh (hunk 2 above)
Comment (platfowner) [reply start_node_docker.sh:8]: Now we can use the default value from blockchain-configs/testnet-prod/node_params.json as we have one now.

File: start_node_incremental_gcp.sh (hunk 3 above)
Comment (platfowner) [reply start_node_incremental_gcp.sh:129]: If env variables are not set explicitly, default values form node_params.json are used instead.

**Comment by platfowner** [comment]
Thanks for the review. Now merging..

File: blockchain-configs/testnet-prod/node_params.json (hunk 1 above)
Comment (ehgmsdk20) [reply blockchain-configs/testnet-prod/node_params.json:71]: Oh, i got it. Thanks for kindly explaining this!

File: start_node_docker.sh (hunk 2 above)
Comment (ehgmsdk20) [reply start_node_docker.sh:8]: I missed the blockchain-configs/testnet-prod/node_params.json, thanks!

**Pull request merged**